shieldlib/src/shield.ts

```diff
@@ -108,23 +113,23 @@ function drawShieldText(r, ctx, shieldDef, routeDef, shieldBounds) {
   ctx.fillStyle = textColor(shieldDef);
   ShieldText.renderShieldText(r, ctx, routeDef.ref, textLayout);
 
-  if (r.options.SHIELD_TEXT_BBOX_COLOR) {
-    ctx.strokeStyle = r.options.SHIELD_TEXT_BBOX_COLOR; //TODO move to debugOptions
+  if (r.debugOptions?.shieldTextBboxColor) {
+    ctx.strokeStyle = r.debugOptions.shieldTextBboxColor; //TODO move to debugOptions
```

Comment:
Now that this is moved to debugOptions, can we get the debug options to work again?

Reply:
![image](https://github.com/ZeLonewolf/openstreetmap-americana/assets/3254090/a04f163b-990f-431b-ab5f-f6c0692206f4)

A lot easier now in TS where I can figure out what's wrong :smile:

Reply:
Fantastic! It's working for me on the main map view, but not on shieldtest.html - any idea why?

Reply:
Fixed in bd0993f!